system_metrics_collector/src/system_metrics_collector/linux_cpu_measurement_node.cpp

```diff
@@ -17,29 +17,30 @@
 
 #include "linux_cpu_measurement_node.hpp"
 
-/* static */ constexpr const char LinuxCpuMeasurementNode::PROC_STAT_FILE[];
-/* static */ constexpr const char LinuxCpuMeasurementNode::CPU_LABEL[];
-/* static */ constexpr const size_t LinuxCpuMeasurementNode::CPU_LABEL_LENGTH;
-
-LinuxCpuMeasurementNode::LinuxCpuMeasurementNode(
-  const std::string & name,
-  const std::chrono::milliseconds measurement_period,
-  const std::string & topic,
-  const std::chrono::milliseconds & publish_period)
-: PeriodicMeasurementNode(name, measurement_period, topic, publish_period)
-{}
+namespace
+{
+constexpr const char PROC_STAT_FILE[] = "/proc/stat";
+constexpr const char CPU_LABEL[] = "cpu";
+constexpr const size_t CPU_LABEL_LENGTH = 3;
```

Comment:
you can actually use `sizeof(CPU_LABEL)` to get the length. (this includes null terminator)
https://stackoverflow.com/questions/46100310/how-to-declare-constexpr-c-string